examples/io-csv-datamapping/io_csv.description

```diff
@@ -0,0 +1,4 @@
+// The Ballerina `io` library contains APIs to read/write CSV content from/to a file.<br/><br/>
```

Comment:
```suggestion
// The Ballerina `io` library contains APIs to read/write CSV content from/to a file.<br/><br/>
```
We will not need these <be> tags with the new BBE design.

Reply:
updated in [d522dc4](https://github.com/ballerina-platform/ballerina-distribution/pull/3038/commits/d522dc4616e757b5e8aae484f3fe3ad401ef0249)